WORKSPACE

```diff
@@ -12,24 +12,15 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "895c2b0d1480834808216fb36cd4bf21517954cb966c893ee42421dfefcfd4bc",
-    strip_prefix = "rules_nodejs-1f6d878a9ea3a095291f66e3d1a0f6b4641f5159",
-    url = "https://github.com/bazelbuild/rules_nodejs/archive/1f6d878a9ea3a095291f66e3d1a0f6b4641f5159.zip",
-)
-
-http_archive(
-    name = "build_bazel_rules_typescript",
-    sha256 = "5a31a67550d2a92270599e90a221afe229f6e1863bf2eff22e9750c6ecf45978",
-    strip_prefix = "rules_typescript-2963b55370b21d545d0ac0f30fca9dc74a0f5538",
-    url = "https://github.com/bazelbuild/rules_typescript/archive/2963b55370b21d545d0ac0f30fca9dc74a0f5538.zip",
+    strip_prefix = "rules_nodejs-0.16.8",
+    url = "https://github.com/bazelbuild/rules_nodejs/archive/0.16.8.zip",
 )
 
 # The @angular repo contains rule for building Angular applications
 http_archive(
     name = "angular",
-    sha256 = "c480904802d62ce63a4955fd8679371a0d9620131c1c424c8786429f4e8ac77e",
-    strip_prefix = "angular-7.1.3",
-    url = "https://github.com/angular/angular/archive/7.1.3.zip",
+    strip_prefix = "angular-8.0.0-beta.3",
```

Comment:
Patch version of the rules_typescript 0.23.2 PR landed in Angular. We should be able to go back to the 7.2.x branch here.